.github/CODEOWNERS

```diff
@@ -1,106 +1,586 @@
 
-# Compute
-# PRLabel: %Compute
-/sdk/compute/                                   @bilaakpan-ms @sandido @dkulkarni-ms @haagha @madewithsmiles @MS-syh2qs @grizzlytheodore
+# PRLabel: %EngSys
+/sdk/template/                                  @Azure/azure-sdk-eng
+
+# Smoke tests
+/common/SmokeTests/                             @AlexGhiondea @schaabs @heaths @tg-msft @jsquire
+
+
+# ######## Management Plane ########
+
+/**/*Management*/                               @allenjzhang @m-nash @markcowl @YalinLi0312 @bquantump @nisha-bhatia
+/**/Azure.ResourceManager*/                     @allenjzhang @m-nash @markcowl @YalinLi0312 @bquantump @nisha-bhatia
+
+# Reviewers to double check any API changes
+/sdk/**/api/                                    @KrzysztofCwalina @tg-msft
+
+# Update AutoRest C#
+/eng/CodeGeneration.targets                     @pakrym @ShivangiReja
+
+
+# ######## Eng Sys ######## 
+/eng/           @weshaggard @chidozieononiwu @mitchdenny @danieljurek
+/eng/mgmt/      @allenjzhang @m-nash @markcowl
+/**/tests.yml   @danieljurek
+/**/ci.yml      @mitchdenny
+
+
+# ######## Services ########
+
+# ServiceLabel: %AAD %Service Attention
+#/<NotInRepo>/            @adamedx
+
+# ServiceLabel: %AKS %Service Attention
+#/<NotInRepo>/            @Azure/aks-pm
+
+# ServiceLabel: %Alerts Management %Service Attention
+/sdk/alertsmanagement/Microsoft.Azure.Management.AlertsManagement            @liadtal @yairgil
+
+# ServiceLabel: %ARM %Service Attention
+#/<NotInRepo>/            @armleads-azure
+
+# ServiceLabel: %ARM - Templates %Service Attention
+#/<NotInRepo>/            @armleads-azure
+
+# ServiceLabel: %ARM - Tags %Service Attention
+#/<NotInRepo>/            @armleads-azure
+
+# ServiceLabel: %ARM - Core %Service Attention
+#/<NotInRepo>/           @armleads-azure
+
+# ServiceLabel: %ARM - Managed Applications %Service Attention
+#/<NotInRepo>/            @armleads-azure
+
+# ServiceLabel: %ARM - Service Catalog %Service Attention
+#/<NotInRepo>/            @armleads-azure
+
+# ServiceLabel: %ARM - RBAC %Service Attention
+#/<NotInRepo>/            @armleads-azure
+
+# ServiceLabel: %Advisor %Service Attention
+/sdk/advisor/Microsoft.Azure.Management.Advisor/            @mojayara @Prasanna-Padmanabhan
+
+# ServiceLabel: %Analysis Services %Service Attention
+/sdk/analysisservices/Microsoft.Azure.Management.AnalysisServices/            @athipp @taiwu @minghan
+
+# ServiceLabel: %API Management %Service Attention
+/sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/            @miaojiang
 
-# Service teams
 # PRLabel: %App Configuration
 /sdk/appconfiguration/                          @annelo-msft @AlexanderSher
 
-/sdk/applicationinsights/Microsoft.Azure.App*/  @divya-jay @geneh @alongafni
-/sdk/operationalinsights/Microsoft.Azure.Ope*/  @divya-jay @geneh @alongafni
+# ServiceLabel: %App Services %Service Attention
+#/<NotInRepo>/            @antcp @AzureAppServiceCLI
 
+# ServiceLabel: %Attestation %Service Attention
```

Comment:
I'd consider removing `%Service Attention` duplication in this file and have the tool that sets up the rule add this label automatically.

Reply:
Talked about this offline -- we'll leave this in for now.